### PR TITLE
Update acceptCard.py

### PR DESCRIPTION
**Pull request by klunkerpunker**
See #coding-disscussion

### CODE REVIEW

File: acceptCard.py [hunk 1]
```diff
@@ -74,3 +74,78 @@ async def acceptCard(
     if newCard:
         cardSheetUnapproved.update_cell(dbRowIndex, 1, cardName)
         cardSheetUnapproved.update_cell(dbRowIndex, 3, authorName)
+
+async def acceptVetoCard(
```
Comment (bones-bones) [inline acceptCard.py:78]: maybe name it vetoAcceptedCard?

File: acceptCard.py [hunk 2]
```diff
@@ -74,3 +74,78 @@ async def acceptCard(
...
+
+    async for message in cardListChannel.history(limit=None):
+        if message.content == cardMessage:
+            try:
+                await message.delete()  # Delete message if it matches
+                print(f"Deleted message: {message.content}")
+            except discord.Forbidden:
+                print("Bot lacks permissions to delete messages.")
+            except discord.HTTPException as e:
+                print(f"Failed to delete message: {e}")
```
Comment (bones-bones) [inline acceptCard.py:112]: Holy shit error handling!?

File: acceptCard.py [hunk 3]
```diff
@@ -74,3 +74,78 @@ async def acceptCard(
...
+
+    async for message in vetoCardListChannel.history(limit=None):
```
Comment (bones-bones) [inline acceptCard.py:114]: Why does it delete it from the veto card list?

File: acceptCard.py (hunk 1 above)
Comment (klunkerpunker) [reply acceptCard.py:78]: This is the thing Liu asked for where the vetoed cards go into the graveyard so I figured it was like "accepting" a vetoed card

File: acceptCard.py (hunk 3 above)
Comment (klunkerpunker) [reply acceptCard.py:114]: I assume just so that there aren't duplicates of old versions in #card-list-corrrupted

File: acceptCard.py (hunk 2 above)
Comment (klunkerpunker) [reply acceptCard.py:112]: I just learned how to do it recently, I could honestly go through and add some error handling if you want

File: acceptCard.py (hunk 3 above)
Comment (klunkerpunker) [reply acceptCard.py:114]: I was just following Liu's instructions

**Review by bones-bones — APPROVED** [review]

**Pull request closed without merging**